lib/dynosaur/process/local.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../process'
+
+module Dynosaur
+  class Process
+    class Local < Process
+      def start
+        pid = ::Process.spawn(rake_command)
+        ::Process.detach(pid)
```

Comment:
If we detach a child process, then this program exits entirely, does the child remain and close properly? The `detach` docs say it sets up another Ruby thread to watch the child, so I'm worried something will zombie.